test_framework/scripts/longhorn-uninstall-setup.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+set -x
+
+run_uninstall_longhorn_test(){
+
+  LONGHORN_TESTS_CUSTOM_IMAGE=${LONGHORN_TESTS_CUSTOM_IMAGE:-"longhornio/longhorn-manager-test:master-head"}
```

Comment:
Just one last doubt, the custom image should come from the specified branch in the jenkins param, right?

Reply:
Hi @khushboo-rancher , updated!